microservices/Keli-Katheya-User-Registration/src/main/java/com/kelikatheya/microservices/userRegistration/security/UserSecurityConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.kelikatheya.microservices.userRegistration.security;
+
+import com.kelikatheya.microservices.userRegistration.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.AutoConfigureOrder;
+import org.springframework.boot.autoconfigure.couchbase.CouchbaseProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+@Configuration
+@EnableWebSecurity
+public class UserSecurityConfig extends WebSecurityConfigurerAdapter {
+    @Autowired
+    BCryptPasswordEncoder bCryptPasswordEncoder;
+
+    @Autowired
+    UserService userService;
+
+    @Autowired
+    Environment environment;
+
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        //cross site request forgery since we use JWT token for authorize user
+        http.csrf().disable();
```

Comment:
bettet to enable csrf as well, JWT is susceptible to XSS. We can discuss

Reply:
if I remove this line, requests will not be proceed and there is no method as enable()

Reply:
By default its enabled. Basically we need one endpoint (we can add it as exception) which gets this token and rest should make requests with it. But lets leave it for now